Gekko/UI/QRCodePickerViewController.swift

```diff
@@ -81,7 +83,56 @@ class QRCodePickerViewController : UIViewController,
             }
         }
     }
-
+    
+    // MARK: Internal methods
+    
+    fileprivate func setupTopBar() {
+        let topBar = UIView()
+        topBar.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
+        topBar.alpha = 0.6
+        
+        view.addSubview(topBar)
+        
+        topBar.snp.makeConstraints { (make) in
+            make.top.equalToSuperview()
+            make.left.equalToSuperview()
+            make.width.equalToSuperview()
+            make.height.equalTo(60)
+        }
+        
+        let label = UILabel()
+        label.text = NSLocalizedString("Scan", comment: "TopBar label")
+        label.font = UIFont.systemFont(ofSize: 24)
```

Comment:
24 seems to be very large.
There is UIDefaults.LabelDefaultFontSize constant. Please try it